packages/framework/block-std/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './clipboard/index.js';
 export * from './command/index.js';
+export * from './edgeless/index.js';
```

Comment:
I'd recommend only exporting the necessary types here, since there are types like `IHitTestOptions` that are not supposed to be reused by `block-std` users.